readme.md

```diff
@@ -194,6 +194,13 @@ Show the system `Services` submenu when right-clicking text on macOS.
 
 Note: Due to [a bug in the Electron implementation](https://github.com/electron/electron/issues/18476), this menu is not identical to the "Services" submenu in the context menus of native apps. Instead, it looks the same as the "Services" menu in the main App Menu. For this reason, it is currently disabled by default.
 
+#### showSelectAll
+
+Type: `boolean`\
+Default: `true`
```

Comment:
This is not the same as in the TypeScript type docs.

Reply:
Sorry thanks, updated. Working on squashing the commits now.